src/js/view/logo.js

```diff
@@ -84,7 +84,13 @@ export default function Logo(_model) {
         _img.src = _settings.file;
 
         const logoInteractHandler = new UI(_logo);
-        logoInteractHandler.on('click tap', function (evt) {
+
+        if (_settings.link) {
+            _logo.setAttribute('tabindex', '0');
+            _logo.setAttribute('aria-label', 'Logo');
```

Comment:
even if the logo does not have a redirect, would we want the user to know it's there, for branding? Branding is one of the reasons for having a logo on your video. I could go either way on this, @aliciahurst any thoughts ?

Reply:
@karimJWP that was part of the acceptance criteria